pyproject.toml

```diff
@@ -16,6 +16,10 @@ pandas = "^1.5.3"
 datasets = "^2.14.0"
 langdetect = "^1.0.9"
 deprecation = "^2.1.0"
+pyvespa = "^0.37.1"
+pyyaml = "^6.0.1"
+sentence-transformers = "^2.2.2"
+torch = "2.0.0"
```

Comment:
this will be fun when we come to upgrade the backend... that's currently pinned to `1.13.0` but maybe we can rip out all the embeddings code